drivers/adc/ad4170/ad4170.h

```diff
@@ -0,0 +1,1142 @@
+#define AD4170_REG_POWER_DOWN_SW		(AD4170_R2B | 0x6e)
+#define AD4170_REG_ADC_CTRL			(AD4170_R2B | 0x70)
+#define AD4170_REG_ERROR_EN			(AD4170_R2B | 0x72)
+#define AD4170_REG_ERROR			(AD4170_R2B | 0x74)
+#define AD4170_REG_INFO1			(AD4170_R2B | 0x76) /* TODO: implement this when it's specified in doc. */
+#define AD4170_REG_CHANNEL_EN			(AD4170_R2B | 0x78)
+#define AD4170_REG_ADC_CHANNEL_SETUP(ch)	(AD4170_R2B | (0x80 + 4 * (ch)))
+#define AD4170_REG_ADC_CHANNEL_MAP(ch)		(AD4170_R2B | (0x82 + 4 * (ch)))
+#define AD4170_REG_ADC_SETUPS_MISC(n)		(AD4170_R2B | (0xc0 + 14 * (n)))
+#define AD4170_REG_ADC_SETUPS_AFE(n)		(AD4170_R2B | (0xc2 + 14 * (n)))
+#define AD4170_REG_ADC_SETUPS_FILTER(n)		(AD4170_R2B | (0xc4 + 14 * (n)))
+#define AD4170_REG_ADC_SETUPS_FILTER_FS(n)	(AD4170_R2B | (0xc6 + 14 * (n)))
+#define AD4170_REG_ADC_SETUPS_OFFSET(n)		(AD4170_R3B | (0xc8 + 14 * (n)))
+#define AD4170_REG_ADC_SETUPS_GAIN(n)		(AD4170_R3B | (0xcb + 14 * (n)))
+#define AD4170_REG_REF_CONTROL			(AD4170_R2B | 0x130)
+#define AD4170_REG_V_BIAS			(AD4170_R2B | 0x134)
+#define AD4170_REG_I_PULLUP			(AD4170_R2B | 0x136)
+#define AD4170_REG_CURRENT_SOURCE(n)		(AD4170_R2B | (0x138 + 2 * (n)))
+#define AD4170_REG_FIR_CONTROL			(AD4170_R2B | 0x140)
+#define AD4170_REG_COEFF_WRITE_DATA		(AD4170_R4B | 0x143)
+#define AD4170_REG_COEFF_READ_DATA		(AD4170_R4B | 0x147)
+#define AD4170_REG_COEFF_ADDRESS		(AD4170_R2B | 0x14b)
+#define AD4170_REG_COEFF_WRRD_STB		(AD4170_R2B | 0x14d)
+#define AD4170_REG_DAC_SPAN			(AD4170_R2B | 0x150)
+#define AD4170_REG_DAC_CHANNEL_EN		(AD4170_R2B | 0x152)
+#define AD4170_REG_DAC_HW_TOGGLE_MASK		(AD4170_R2B | 0x154)
+#define AD4170_REG_DAC_HW_LDAC_MASK		(AD4170_R2B | 0x156)
+#define AD4170_REG_DAC_DATA(ch)			(AD4170_R2B | (0x158 + 2 * (ch)))
+#define AD4170_REG_DAC_SW_TOGGLE_TRIGGERS	(AD4170_R2B | 0x168)
+#define AD4170_REG_DAC_SW_LDAC_TRIGGERS		(AD4170_R2B | 0x16a)
+#define AD4170_REG_DAC_INPUTA(ch)		(AD4170_R2B | (0x16c + 2 * (ch)))
+#define AD4170_REG_DAC_INPUTB(ch)		(AD4170_R2B | (0x17c + 2 * (ch)))
+#define AD4170_REG_GPIO_MODE			(AD4170_R2B | 0x190)
+#define AD4170_REG_OUTPUT_DATA			(AD4170_R2B | 0x192)
+#define AD4170_REG_INPUT_DATA			(AD4170_R2B | 0x194)
+
+/* AD4170_REG_INTERFACE_CONFIG_A */
+#define AD4170_SW_RESET_MSK					NO_OS_BIT(7)
+#define AD4170_ADDR_ASCENSION_MSK				NO_OS_BIT(5)
+#define AD4170_SDO_ENABLE_MSK					NO_OS_BIT(4)
+#define AD4170_SW_RESETX_MSK					NO_OS_BIT(0)
+
+/* AD4170_REG_INTERFACE_CONFIG_B */
+#define AD4170_INTERFACE_CONFIG_B_SINGLE_INST_MSK		NO_OS_BIT(7)
+#define AD4170_INTERFACE_CONFIG_B_SHORT_INSTRUCTION_MSK		NO_OS_BIT(3)
+#define AD4170_INTERFACE_CONFIG_B_SHORT_INSTRUCTION(x)		(((x) & 0x1) << 3)
+
+/* AD4170_REG_INTERFACE_CONFIG_C */
+#define AD4170_INTERFACE_CONFIG_C_CRC_MSK			(NO_OS_BIT(7) | NO_OS_BIT(6) | NO_OS_BIT(1) | NO_OS_BIT(0))
+#define AD4170_INTERFACE_CONFIG_C_CRC(x)			(((~x) & 0x3) | (((x) << 6) & 0xC0))
+#define AD4170_INTERFACE_CONFIG_C_STRICT_REG_ACCESS_MSK	NO_OS_BIT(5)
```

Comment:
Pay attention to white spaces and alignment.

Reply:
will do